### DISCUSSION TITLE
Welcome to markdown-text Discussions!

### DISCUSSION THREAD

**Discussion by pncsoares** [from-zero-to-fullstack/markdown-text#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pncsoares** [comment]
@ma-oliveiramarques o ficheiro markdown que criaste tem algumas coisas erradas.

Um título é com # e não com *

`# título` 👇

# título

`**texto a negrito**` 👇

**texto a negrito**

Não é boa prática colocar linhas por baixo de títulos através de ===

Podes pf validar o ficheiro, alterar e lançar novo Pull Request?

Obrigado 😊

**Reply by ma-oliveiramarques** [reply]
Obrigada!
De qualquer forma, eu tentei fazer o título de acordo com esta parte do Markdown: https://www.markdownguide.org/basic-syntax/#alternate-syntax
